agentstack/templates/langgraph/{{cookiecutter.project_metadata.project_slug}}/README.md

```diff
@@ -0,0 +1,37 @@
+# {{ cookiecutter.project_metadata.project_name }}
+{{ cookiecutter.project_metadata.description }}
+
+## How to build your Crew
```

Comment:
Need to update these docs to reference LangGraph

Reply:
done